deps: bump logrus library version

asg0451 opened the pull request:
Bump logrus library version to pull in vulnerability fix.

Epic: None

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/139601)
<!-- Reviewable:end -->

asg0451 commented:
this fails with the following error which i can reproduce. i'm not sure what it means though or how to resolve it. @rickystewart any idea?

```
ERROR: /private/var/tmp/_bazel_miles/8e6f37a11f54ea4e6a58bb92f8ed964e/external/go_googleapis/google/api/BUILD.bazel: no such target '@@go_googleapis//google/api:httpbody_go_proto': target 'httpbody_go_proto' not declared in package 'google/api' defined by /private/var/tmp/_bazel_miles/8e6f37a11f54ea4e6a58bb92f8ed964e/external/go_googleapis/google/api/BUILD.bazel (did you mean httpbody_proto, or httpbody.proto?)
ERROR: /private/var/tmp/_bazel_miles/8e6f37a11f54ea4e6a58bb92f8ed964e/external/com_github_grpc_ecosystem_grpc_gateway_v2/runtime/BUILD.bazel:5:11: no such target '@@go_googleapis//google/api:httpbody_go_proto': target 'httpbody_go_proto' not declared in package 'google/api' defined by /private/var/tmp/_bazel_miles/8e6f37a11f54ea4e6a58bb92f8ed964e/external/go_googleapis/google/api/BUILD.bazel (did you mean httpbody_proto, or httpbody.proto?) and referenced by '@@com_github_grpc_ecosystem_grpc_gateway_v2//runtime:runtime'
ERROR: Analysis of target '//pkg/cmd/cockroach-short:cockroach-short' failed; build aborted: Analysis failed
```

msbutler commented on go.mod:
```diff
@@ -37,9 +37,9 @@ require (
-	google.golang.org/genproto v0.0.0-20230526161137-0005af68ea54
-	google.golang.org/grpc v1.57.2
+	google.golang.org/api v0.128.0
+	google.golang.org/genproto v0.0.0-20231012201019-e917dd12ba7a
```
woah, is it necessary to update these core libraries? see comment above.

asg0451 committed:
deps: bump logrus library version

Bump logrus library version to pull in vulnerability fix.

Epic: None

Release note: None
